.github/workflows/build_test_deploy.yml

```diff
@@ -1173,6 +1172,10 @@ jobs:
 
       - run: echo "${{ steps.docs-chang.outputs.DOCS_CHANGE }}"
 
+      - name: Check codemod only change
+        run: echo "CODEMOD_CHANGE<<EOF" >> $GITHUB_OUTPUT; echo "$(node scripts/run-for-change.js --type next-codemod --always-canary --exec echo 'yup')" >> $GITHUB_OUTPUT; echo "EOF" >> $GITHUB_OUTPUT
+        id: codemod-change
```

Comment:
Is this meant to be under the `build` job?

Reply:
Yep 🤦‍♂️